website/blog/2020-09-19-lithium.md

```diff
@@ -0,0 +1,266 @@
+---
+author: Tomasz Godzik
+title: Metals v0.9.3 - Lithium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.9.4, which focused on improving
+the Scala 3 worksheets as well as a number of smaller improvements.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">105</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">44</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">6</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">15</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">2</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/27?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Scala 0.26.0 and 0.27.0-RC1 support.
+- Analyze stacktrace command
+- Customizable package exclusions
+
+## Analyze stacktrace command
+
+Thanks to [kpbochenek](https://github.com/kpbochenek) Metals can now help with
+navigating stack traces using the new `metals.analyze-stacktrace` command. The
+only thing it needs is the actual stack trace string to be input as a parameter
+to the command. This new functionality works differently in Visual Studio Code
+and other supported editors.
+
+- for Visual Studio Code
+
+The only thing users need to do is copy the stack trace and run the
+`Analyze Stacktrace` command from the command palette. This will display a new
+web view, where it will be possible for users to navigate the files that are
+included in the exception.
+
+![stacktrace-vscode](https://i.imgur.com/WBU4hvT.gif)
+
+- for other editors
+
+To make it work with any other editors users need to run the command manually
+with the stacktrace as input. The command will generate a new file inside
+`.metals/stacktrace.scala` which can be used to navigate throught the stacktrace
+using code lenses.
```

Comment:
```suggestion
To make it work with any other editors users need to run the command manually
with the stacktrace as input, similar to the example above with `coc-metals`. The command will generate a new file, `.metals/stacktrace.scala`, which can be used to navigate through the stacktrace using code lenses.
```

Since we can now have a coc-metals section, we can change the wording up here a bit.